audio/stereo_panner.rs

```diff
@@ -0,0 +1,132 @@
+use block::{Block, Chunk, Tick, FRAMES_PER_BLOCK};
+use euclid::Vector3D;
+use node::{AudioNodeEngine, AudioNodeMessage, BlockInfo};
+use node::{AudioNodeType, ChannelInfo};
+use param::{Param, ParamDir, ParamType};
+use std::f32::consts::PI;
+
+// .normalize(), but it takes into account zero vectors
+use panner_node::normalize_zero;
+
+#[derive(Copy, Clone, Debug)]
+pub struct StereoPannerOptions {
+    pub pan: f32,
+}
+
+impl Default for StereoPannerOptions {
+    fn default() -> Self {
+        StereoPannerOptions {
+            pan: 0.,
+        }
+    }
+}
+
+#[derive(AudioNodeCommon)]
+pub(crate) struct StereoPannerNode {
+    channel_info: ChannelInfo,
+    pan: Param,
+}
+
+impl StereoPannerNode {
+    pub fn new(options: StereoPannerOptions, channel_info: ChannelInfo) -> Self {
+        Self {
+            Param::new(options.pan),
+        }
+    }
+
+    pub fn update_parameters(&mut self, info: &BlockInfo, tick: Tick) -> bool {
+        self.pan.update(info, tick)
+    }
+}
+
+impl AudioNodeEngine for StereoPannerNode {
+    fn node_type(&self) -> AudioNodeType {
+        AudioNodeType::StereoPannerNode
+    }
+
+    fn process(&mut self, mut inputs: Chunk, info: &BlockInfo) -> Chunk {
+        debug_assert!(inputs.len() == 1);
+
+        return inputs;
```

Comment:
Just pointing out that the rust compiler should be complaining about a lot of unreachable code after this line.

Reply:
This line still needs to be removed